package.json

```diff
@@ -1,67 +1,49 @@
 {
-  "name": "be-the-best",
-  "version": "0.1.0",
-  "private": true,
-  "dependencies": {
-    "@ethersproject/providers": "^5.6.2",
-    "@testing-library/jest-dom": "^5.16.3",
-    "@testing-library/react": "^12.1.4",
-    "@testing-library/user-event": "^13.5.0",
-    "@types/jest": "^27.4.1",
-    "@types/node": "^16.11.26",
-    "@types/react": "^17.0.43",
-    "@types/react-dom": "^17.0.14",
-    "@web3-react/core": "^6.1.9",
-    "@web3-react/injected-connector": "^6.0.7",
-    "ethers": "^5.6.2",
-    "react": "^18.0.0",
-    "react-dom": "^18.0.0",
-    "react-scripts": "5.0.0",
-    "typescript": "^4.6.3",
-    "web-vitals": "^2.1.4"
-  },
-  "devDependencies": {
-    "@typescript-eslint/eslint-plugin": "^5.17.0",
-    "@typescript-eslint/parser": "^5.17.0",
-    "eslint": "^8.12.0",
-    "eslint-plugin-react": "^7.29.4",
-    "eslint-plugin-react-hooks": "^4.4.0",
+	"name": "be-the-best-front",
+	"version": "0.1.0",
+	"private": true,
+	"scripts": {
+		"dev": "next dev",
+		"build": "next build",
+		"start": "next start",
+		"lint": "next lint"
```

Comment:
moze dodamy jeszcze jakis scrypt na puszczenie testow. Wczesniej mielismy react-scripts test

Reply:
nie mamy aktualnie testów. jak skonfigurujemy testy to będzie sens dodać skrypt do ich uruchamiania :)